### DISCUSSION TITLE
Can I install a previous version?

### DISCUSSION THREAD

**Discussion by darrylhallcatman** [aclap-dev/video-downloadhelper#186] [Help and Q&A]
### Your question

The current version has a bug.  There is something wrong with the deb 64 bit file.  Neither Mint nor Makulu seem to notice that it has been installed upon which the user is prompted to install (again).
The sign up page required by github leads to a sign in page instead.

### Addon & CoApp version and details

The downloader package is Version 8.2.0.23
I couldn't find the coapp version but did write down the deb filename.  The filename is vdhcoapp-linux-x86

**Comment by paulrouget** [comment]
Yes, here: https://github.com/aclap-dev/vdhcoapp/releases

But can you please help out. What's the issue exactly with the 64 bit file?

**Reply by darrylhallcatman** [reply]
My days of diagnosing bugs ended in the early ninetees.  All I can tell you is that Linux Mint doesn't seem to see it after it is installed.

**Comment by paulrouget** [comment]
Also, please read this: https://github.com/aclap-dev/video-downloadhelper/wiki/CoApp-not-recognized - it might fix your issue.
